config/di.php

```diff
@@ -66,9 +66,13 @@ function (Container $container) {
         function (Container $container) {
             $cleanRequestAttributesMiddleware = new \Sainsburys\HttpService\Components\Middlewares\MiddlewareLibrary\BeforeMiddleware\CleanRequestAttributes();
             $convertToJsonResponseObjectMiddleware = new \Sainsburys\HttpService\Components\Middlewares\MiddlewareLibrary\BeforeMiddleware\ConvertToJsonResponseObject();
+            $unpackJsonParametersToArrayMiddleware = new \Sainsburys\HttpService\Components\Middlewares\MiddlewareLibrary\BeforeMiddleware\UnpackJsonParametersToArray();
+
             $middlewareManager = new \Sainsburys\HttpService\Components\Middlewares\MiddlewareManager();
             $middlewareManager->addToEndOfBeforeMiddlewareList($cleanRequestAttributesMiddleware);
             $middlewareManager->addToEndOfBeforeMiddlewareList($convertToJsonResponseObjectMiddleware);
+            $middlewareManager->addToEndOfBeforeMiddlewareList($unpackJsonParametersToArrayMiddleware);
+
             return $middlewareManager;
```

Comment:
@pharman I agree, the coupling is a bit too tight, ideally which framework middlewares the service wants/needs to use should be user defined. Would adding the middleware in this PR to the middleware-manager in index.php (like service-proxy does) be better ?